lib/components/chat_history/twitch/message.dart

```diff
@@ -26,6 +27,13 @@ const colors = [
   Color(0xFF00FF7F),
 ];
 
+class _MessageLink {
```

Comment:
This class doesn't seem necessary? `hasLink` isn't used anywhere so `url` can be passed directly.